tests/test_models/test_user.py

```diff
@@ -127,6 +127,6 @@ def test_to_dict_values(self):
 
     def test_str(self):
         """test that the str method has the correct output"""
-        user = User()
-        string = "[User] ({}) {}".format(user.id, user.__dict__)
-        self.assertEqual(string, str(user))
+        u = User()
+        string = "[User] ({}) {}".format(u.id, u.__dict__)
+        self.assertEqual(string, str(u))
```

Comment:
Many of the same issues as City. Rinse and repeat.

Reply:
Roger that!